### DISCUSSION TITLE
Unable to login via email

### DISCUSSION THREAD

**Discussion by MAbdurrehman13** [vercel/community#7415] [Bug Report] [answered]
### Page to Investigate

https://vercel.com/login

### Steps to Reproduce

Click `Continue with Email`
<img width="417" alt="image" src="https://github.com/vercel/community/assets/174642520/3f6b82c9-e6ab-4beb-8fa8-a2649c0037c2">

Add your email and login
<img width="449" alt="image" src="https://github.com/vercel/community/assets/174642520/66cfaada-bf9b-47a0-93a5-d911a97a1afe">

email is being passed but still a 400 with this msg

Request:
<img width="982" alt="image" src="https://github.com/vercel/community/assets/174642520/1010c192-ea3a-4671-b2f7-1379ca72e467">

Response:
<img width="927" alt="image" src="https://github.com/vercel/community/assets/174642520/46f29714-dd65-4253-b485-90a3692a564a">

**Comment by pawlean** [comment, marked as answer]
Hi @MAbdurrehman13!

There was an incident yesterday that caused this, you should be able to login again.

[a comment by an automated account is not shown]
